ext/js/language/language-transformer.js

```diff
@@ -133,7 +135,9 @@ export class LanguageTransformer {
         const results = [this._createTransformedText(sourceText, 0, [])];
         for (let i = 0; i < results.length; ++i) {
             const {text, conditions, trace} = results[i];
-            for (const {name, rules} of this._transforms) {
+            for (const {name, rules, suffixHeuristic} of this._transforms) {
```

Comment:
Out of curiosity, is there any performance difference if we don't destructure everything up-front? But instead do something like
```js
for (const transform of this._transforms) {
  if (!transform.suffixHeuristic.test(text)) { continue; }
  const {name, rules} = transform;
  ...
}

Reply:
It seems that this actually improves performances by a small margin